### ISSUE TITLE
Issue with install 

### ISSUE THREAD

**Issue by killmasta93** [asbru-cm/asbru-cm#129]
**Describe the bug**
after putting the repository cannot install it

**To Reproduce**
curl -s https://packagecloud.io/install/repositories/asbru-cm/asbru-cm/script.deb.sh | sudo bash
Detected operating system as Ubuntu/cosmic.
Checking for curl...
Detected curl...
Checking for gpg...
Detected gpg...
Running apt-get update... done.
Installing apt-transport-https... done.
Installing /etc/apt/sources.list.d/asbru-cm_asbru-cm.list...done.
Importing packagecloud gpg key... done.
Running apt-get update... done.

then ran this 
apt-get install asbru-cm
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package asbru-cm

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
Kubuntu 18.04

**Additional context**

**Comment by gfrenoy** [comment]
> Detected operating system as Ubuntu/cosmic.

Looks like a duplicate of #113, isn't it ?

**Comment by killmasta93** [comment]
Thanks for the reply, i was looking at open request and found that request but every time i open i get this 
![screenshot at 2019-02-15 15-42-24](https://user-images.githubusercontent.com/13953629/52890387-7619ea80-3152-11e9-9c18-7747fb872f8c.png)

**Comment by gfrenoy** [comment]
This message is shown only once.  Actually, it is shown if the file ```pac_notray.notified``` does not exist in your configuration directory (typically ```~/config/.pac```).

**Comment by killmasta93** [comment]
thanks for the reply, did not know thank you again everything is working wonderful. as our of curiosity on the terminal to copy says control +shitf+C is there a way i can put just control + C?

**Comment by gfrenoy** [comment]
See #115 

Please also stay on topic ; for chatting, feel free to join the [IRC room](https://webchat.st-city.net/#asbru-cm).
